lablup/backend.ai-webui
# feat: add `allowAppDownloadPanel` that can hide app download panel

resolves #1979 
If `allowAppDownloadPanel` is true, display the webui app download panel on the summary page. Or, hide it.

How to test: 
1. set `allowAppDownloadPanel` to the config.toml
2. check the app download panel is hidden/shown according to the option.

**Checklist:** (if applicable)

- [x] Minium required manager version : 22.09
